planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/Optimizer.java

```diff
@@ -28,25 +28,28 @@
    private static Logger log = LoggerFactory.getLogger(Optimizer.class);
    private final int NUMBER_OF_PLANS_GENERATED;
    private final SearchMethodName searchName;
+   private final double HYSTERESIS_PROPORTION; 
 
    public Optimizer() {
-      NUMBER_OF_PLANS_GENERATED = 5;
-      searchName = SearchMethodName.BLINDSEARCH;
+      this(5,SearchMethodName.BLINDSEARCH,0.5);
```

Comment:
It could be better using Constants, for example `DAFAULT_NUMBER_OF_PLANS_GENERATED = 5`